Bylaws.md

```diff
@@ -375,38 +393,40 @@ Date: \_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
-    * Threatening violence towards anyone.  
+ 2. In connection with official Forum activities, all Forum participants shall refrain from conduct such as:
+    * Threatening violence towards anyone.
     * Discriminating against anyone on the basis of personal characteristics or group membership.
     * Harassing or bullying anyone verbally, physically, or sexually.
-    * Launching barbs at others.  [Note:  a "barb" is an obviously or openly unpleasant or carping remark.]
+    * Launching barbs at others. [Note: a "barb" is an obviously or openly unpleasant or carping remark.]
     * Touching another person in a physically inappropriate way. 
     * Deliberately intimidating or stalking another person (in-person, online, or by other means).
-    * Inappropriately disrupting or impeding official Forum events, including meetings, talks, and presentations.  For purposes of this Code, "inappropriate disruption" would include aggressive, violent, and abusive conduct that prevents an official Forum event from occurring or proceeding.
+    * Inappropriately disrupting or impeding official Forum events, including meetings, talks, and presentations. For purposes of this Code, "inappropriate disruption" would include aggressive, violent, and abusive conduct that prevents an official Forum event from occurring or proceeding.
     * Spamming, trolling, flaming, baiting, and other similar behavior inappropriately directed towards an individual.
+    * Lying, deceiving, or intentionally misleading by omission or half-truth.
     * Advocating for, or encouraging, any of the above behavior.
 
  3. All Forum participants should promote the rules of this Code and take action to bring discussions back into compliance with the Code whenever violations are observed.
 
  4. Forum participants should stick to ideological, conceptual discussions and avoid engaging in offensive or sensitive personal discussions, particularly if they're off-topic; such personal discussions can lead to unnecessary arguments, hurt feelings, and damaged trust. 
 
-**II.  Moderation.**  These are the policies for upholding the Code.
- 1. Resist the urge to be defensive.  Remember that it's your responsibility to clearly communicate your message to your fellow participants. Everyone wants to get along and we are all in the Forum first and foremost because we want to talk about standards and everything that involves. Other participants will be eager to assume good intent and forgive as long as you have earned their trust.
+**II. Moderation.** These are the policies for upholding the Code.
+ 1. Resist the urge to be defensive. Remember that it's your responsibility to clearly communicate your message to your fellow participants. Everyone wants to get along and we are all in the Forum first and foremost because we want to talk about standards and everything that involves. Other participants will be eager to assume good intent and forgive as long as you have earned their trust.
```

Comment:
Nit: 1. and 2. are not formatted as items within the list.

Reply:
Fixed but please check again in case the rendering is not the one expected.

Reply:
LGTM